app/panel/components/Blocking/Category.jsx

```diff
@@ -153,6 +157,30 @@ class Category extends React.Component {
 		}
 	}
 
+	_renderCaret() {
+		const { isExpanded } = this.state;
+		const { isUnknown } = this.props;
+		// const caretClasses = (this.state.isExpanded ? 'caret-up' : 'caret-down') + (isUnknown ? ' Category__antiTrackingCaret' : '');
+		const caretClasses = ClassNames(this.context, {
+			'caret-down': isExpanded,
+			'caret-up': !isExpanded,
+			Category__antiTrackingCaret: isUnknown
+		});
+		if (isExpanded) {
+			return (
+				<svg className={caretClasses} onClick={this.toggleCategoryTrackers} width="11" height="7" viewBox="0 0 11 7" xmlns="http://www.w3.org/2000/svg">
+					<path d="M1.21.02L-.02 1.25l5.27 5.27 5.27-5.27L9.29.02 5.25 4.06" fillRule="evenodd" />
+				</svg>
+			);
+		}
+		return (
+			<svg className={caretClasses} onClick={this.toggleCategoryTrackers} width="11" height="7" viewBox="0 0 11 7" xmlns="http://www.w3.org/2000/svg">
+				<path d="M1.283 7L0 5.676 5.5 0 11 5.676 9.717 7 5.5 2.649z" fillRule="evenodd" />
+			</svg>
+		);
+		// 4a4a4a
```

Comment:
You only need to change the `<path />` element's attributes here depending on the `isExpanded` bool, so the ternary can go inside of the `<svg />` tags to avoid repeat code.

Also, the `// 4a4a4a` comment looks like it can be removed.

Reply:
I didn't notice the SVG's were that similar! Good refactor, also removed the 4a4a4a (sorry!)